authors/apps/notifications/views.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework import status
+from authors.apps.notifications.models import Notification, Notifier
+from authors.apps.notifications.serializers import NotificationSerializer
+from rest_framework.permissions import IsAuthenticated
+
+class NotificationViews(APIView):
+    
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):
+        current_user = request.user
+        notifications = Notification.objects.filter(receiver=current_user)
+        serializer = NotificationSerializer(notifications, many=True)
+        return Response(
+            {
+                "notifications_count": len(notifications),
+                "notifications": serializer.data
+            },
+            status.HTTP_200_OK
+        )
+
+    def post(self, request):
+        current_user = request.user
+        notifier = Notifier.objects.filter(user=current_user).first()
+        if notifier:
+            if notifier.status==True:
+                notifier.status = False
```

Comment:
Please consider using a ternary operator for these two lines.
This is the format:

`condition ? value_if_true : value_if_false`

For more information, [see this resource](https://guide.freecodecamp.org/c/ternary-operator/)